src/systemd/Makefile.am

```diff
@@ -48,6 +51,7 @@ src/systemd/%: src/systemd/%.in
 systemdgenerated = \
 	$(nodist_systemdunit_DATA) \
 	$(nodist_tempconf_DATA) \
+	$(nodist_sysusers_DATA) \
```

Comment:
I'd lean towards doing that in `configure.ac`.  The only reason we need to do these ones here is to get the recursive path expansion (see the comment just above), but that's not an issue for these files.

Reply:
I was actually vaguely aware of this, but I found it nice to keep all the systemd bits together in one file. I don't have a strong opinion about it, though.